pkgs/applications/terminal-emulators/kitty-themes/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub, lib, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kitty-themes";
+  version = "2021-01-04";
+
+  src = fetchFromGitHub {
+    owner = "kovidgoyal";
+    repo = pname;
+    rev = "fca3335489bdbab4cce150cb440d3559ff5400e2";
```

Comment:
Why are you using such an old commit? https://github.com/kovidgoyal/kitty-themes/commit/fca3335489bdbab4cce150cb440d3559ff5400e2 is from 2019. Use the latest one instead, which is https://github.com/kovidgoyal/kitty-themes/commit/76e0c1151ea2d48a6c9c3842095c9781051e5b88.